Api.js

```diff
@@ -0,0 +1,524 @@
+/**

```

Comment:
This should not be in a pull request. You should pull down master and use that as your base, so file names should not differ.